telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/TelemetryParser.kt

```diff
@@ -86,7 +87,7 @@ object TelemetryParser {
         try {
             schema.validate(JSONObject(fileContents))
         } catch (e: Exception) {
-            System.err.println("Schema validation failed due to thrown exception $e")
+            System.err.println("Schema validation failed due to thrown exception $e\n on input: $fileContents")
```

Comment:
if input is likely to be multiline it should start on its own line.

```suggestion
            System.err.println("Schema validation failed due to thrown exception $e\n on input:\n$fileContents")
```

Reply:
Sure, will change